project-base/symfony.lock

```diff
@@ -1181,11 +1067,5 @@
     },
     "zalas/phpunit-injector": {
         "version": "v1.2.7"
-    },
```

Comment:
those many changes in `symfony.lock` are confusing and a little disturbing :smile: How come there are so many of them? :thinking: all the removed packages were dependencies of the original faker?

Reply:
actually, I just run composer update and commit changes... I assumed it was something previously forgotten... 
I can limit changes in symfony(ies).lock, but the I would like to sync them so we don't have changes after each composer update

Reply:
ah, yeah, that could be the reason :+1: Thanks for the explanation :blush: